pkg/agent/agent.go

```diff
@@ -127,10 +129,13 @@ func InitializeWithAgentFeatures(centralCfg config.CentralConfig, agentFeaturesC
 	if centralCfg.GetUsageReportingConfig().IsOfflineMode() {
 		// Offline mode does not need more initialization
 		agent.cfg = centralCfg
```

Comment:
This can be removed as setCentralConfig sets the variable

Reply:
thank you